app/src/features/surveys/SurveyPermitForm.tsx

```diff
@@ -77,97 +64,109 @@ export const SurveyPermitFormYupSchema = yup.object().shape({
             {values.permit.permits?.map((permit, index) => {
               const permitNumberMeta = getFieldMeta(`permit.permits.[${index}].permit_number`);
               const permitTypeMeta = getFieldMeta(`permit.permits.[${index}].permit_type`);
 
               return (
-                <ListItem disableGutters key={index}>
-                  <ListItemText>
-                    <Box>
-                      <Grid container spacing={2}>
-                        <Grid item sm={6}>
-                          <CustomTextField
-                            name={`permit.permits.[${index}].permit_number`}
-                            label="Permit Number"
-                            other={{
-                              required: true,
-                              value: permit.permit_number,
-                              error: permitNumberMeta.touched && Boolean(permitNumberMeta.error),
-                              helperText: permitNumberMeta.touched && permitNumberMeta.error
-                            }}
-                          />
-                        </Grid>
-                        <Grid item sm={6}>
-                          <FormControl
-                            variant="outlined"
-                            fullWidth
-                            required={true}
-                            error={permitTypeMeta.touched && Boolean(permitTypeMeta.error)}>
-                            <InputLabel id="permit_type">Permit Type</InputLabel>
-                            <Select
-                              id={`permit.permits.[${index}].permit_type`}
-                              name={`permit.permits.[${index}].permit_type`}
-                              labelId="permit_type"
-                              label="Permit Type"
-                              value={permit.permit_type}
-                              onChange={handleChange}
-                              displayEmpty
-                              inputProps={{ 'aria-label': 'Permit Type' }}>
-                              <MenuItem key={1} value="Park Use Permit">
-                                Park Use Permit
-                              </MenuItem>
-                              <MenuItem key={2} value="Wildlife Permit - General">
-                                Wildlife Permit - General
-                              </MenuItem>
-                              <MenuItem key={3} value="Scientific Fish Collection Permit">
-                                Scientific Fish Collection Permit
-                              </MenuItem>
-                            </Select>
-                            <FormHelperText>{permitTypeMeta.touched && permitTypeMeta.error}</FormHelperText>
-                          </FormControl>
-                        </Grid>
-                      </Grid>
-                    </Box>
-                  </ListItemText>
-                  <ListItemSecondaryAction>
+                <Collapse role="listitem" key={index}>
```

Comment:
Should update this key to a non index value.
https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66

Reply:
Ideal yes - for now we'll just leave it as mark for tech debt.